### PR TITLE
Move ACL migration to migrate up command, and force migration

### PR DESCRIPTION
**Pull request by guy-har**

### CODE REVIEW

File: pkg/kv/migration.go [hunk 1]
```diff
@@ -54,6 +54,10 @@ func ValidateSchemaVersion(ctx context.Context, store Store) (int, error) {
 		logging.Default().Info("Migration to KV required. Did you migrate using version v0.80.x? https://docs.lakefs.io/reference/upgrade.html#lakefs-0800-or-greater-kv-migration")
 		return 0, ErrMigrationRequired
 	}
+	if kvVersion < ACLMigrateVersion {
+		logging.Default().Info("Migration to ACL required. Did you migrate using version v0.98.x? https://docs.lakefs.io/reference/access-control-list.html#migrating-from-the-previous-version-of-aclsn")
```
Comment (itaiad200) [inline pkg/kv/migration.go:58]: 💪

File: cmd/lakefs/cmd/migrate.go [hunk 2]
```diff
@@ -162,8 +141,37 @@ var upCmd = &cobra.Command{
...
 		defer kvStore.Close()
 
-		_ = mustValidateSchemaVersion(ctx, kvStore)
-		fmt.Printf("No migrations to apply.\n")
+		force, _ := cmd.Flags().GetBool("force")
+
+		// -- This part contains the migration to ACL and will be removed in next version
+
+		// skip migrate to ACL for users with External authorizations
+		if !cfg.IsAuthTypeAPI() {
```
Comment (itaiad200) [inline cmd/lakefs/cmd/migrate.go:149]: Let's log if we skipped the migration

File: cmd/lakefs/cmd/migrate.go [hunk 3]
```diff
@@ -162,8 +141,37 @@ var upCmd = &cobra.Command{
...
+				return
+			}
+
+			migrateToACL(ctx, kvStore, cfg, logging.Default(), true, now)
+		}
+
+		err = kv.SetDBSchemaVersion(ctx, kvStore, kv.ACLMigrateVersion)
+		if err != nil {
+			// this should not happen
+			fmt.Println("migration succeeded - failed to upgrade version")
```
Comment (itaiad200) [inline cmd/lakefs/cmd/migrate.go:168]: What's the action item here? Should the user rerun?

**Review by itaiad200 — APPROVED** [review]
Approved - see some comments

File: cmd/lakefs/cmd/migrate.go (hunk 3 above)
Comment (guy-har) [reply cmd/lakefs/cmd/migrate.go:166]: Added thanks to printed message.
IMO A better way would be to run `migrate goto --force`, but I didn't want to insert this decision as part of this PR

**Pull request merged**